### PR TITLE
Fix how this SDK identifies itself to the Cloud Dataflow service

### PR DESCRIPTION
**Pull request by davorbonaci**
* Use Beam SDK naming, instead of Dataflow SDK.
* Fix version lookup, so it correctly identifies the version.

R: @lukecwik

### CODE REVIEW

File: sdks/java/core/src/main/java/org/apache/beam/sdk/util/DataflowReleaseInfo.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(DataflowReleaseInfo.class);
 
   private static final String DATAFLOW_PROPERTIES_PATH =
-      "/org.apache.beam/sdk/sdk.properties";
```
Comment (lukecwik) [inline sdks/java/core/src/main/java/org/apache/beam/sdk/util/DataflowReleaseInfo.java]: Please rename file to BeamReleaseInfo or just ReleaseInfo

Comment (davorbonaci) [inline sdks/java/core/src/main/java/org/apache/beam/sdk/util/DataflowReleaseInfo.java]: Done.

**Comment by davorbonaci** [comment]
PTAL.

**Comment by lukecwik** [comment]
LGTM

Would be better with test since would have caught the original break.
Also we could rely on the file always being there and slightly simplify the code.

**Pull request closed without merging**